clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -853,7 +844,7 @@ public void onSuccess(ClientResponse resp, RequestFuture<Void> future) {
 
         @Override
         public void onFailure(RuntimeException e, RequestFuture<Void> future) {
-            clearFindCoordinatorFuture();
+            log.debug("FindCoordinator request failed", e);
```

Comment:
nit: I think it's better to just print the e.message in a single line.

Reply:
You mean like 
```
log.debug("FindCoordinator request failed due to {}", e.getMessage());
```
?

Reply:
if you just have

```
log.debug("FindCoordinator request failed due to {}", e)
```

Then e.toString would be called which would usually be `e.name(): e.getMessage()`.